### ISSUE TITLE
Quick change device attributes

### ISSUE THREAD

**Issue by photkey** [marmig0404/pymemuc#21]
```
15. 描述：一键换机功能(7.5.0 新增)
    语法：memuc randomize <-i vmindex | -n vmname>
    示例：memuc randomize -i 0
    返回：SUCCESS: change device attributes finished.
```
URL：https://bbs.xyaz.cn/thread-365515-1-1.html

Sorry, I'm relying on translations for all my English, but this official document in Chinese, which adds the ability to switch to a new machine with one click, will hopefully be added.

`memuc.memuc_run(["randomize","-i", "1"])` I've just tried this to achieve the same function, but it's a bit tricky to write.

**Comment by marmig0404** [comment]
This would be a good feature to add!
Its interesting it is not documented in the official [memuc documentation](https://www.memuplay.com/blog/memucommand-reference-manual.html), this is what I based the included features on.

If you would like to contribute I think it would go well in `pymemuc/_manage.py`, otherwise I can do it sometime later.

**Comment by marmig0404** [comment]
Here is a good resource for contributing: https://docs.github.com/en/get-started/quickstart/contributing-to-projects

**Comment by marmig0404** [comment]
@photkey 
unrelated but i've added a translation to the documentation, let me know how it is: https://pymemuc.martinmiglio.dev/zh/latest/

if you'd like to contribute with the translations, see `docs/source/locales/zh/*.po`

**Comment by marmig0404** [comment]
this function is now available in v0.2.6 😁
